### PR TITLE
Adding visual cue for "Limited" & "Private" organizations.

### PR DESCRIPTION
**Pull request by spenaustin**
Implements #13017 .

Label for "Limited" & "Private" organizations added to org header, org homepage, and navbar.

Before:

> Screen Shot 2020-10-05 at 4 06 59 PM
> Screen Shot 2020-10-05 at 4 06 49 PM

After:

> Screen Shot 2020-10-05 at 4 06 07 PM
> Screen Shot 2020-10-05 at 4 07 30 PM
> Screen Shot 2020-10-05 at 4 21 25 PM
> Screen Shot 2020-10-05 at 4 21 18 PM
> Screen Shot 2020-10-05 at 4 06 28 PM
> Screen Shot 2020-10-05 at 4 07 20 PM

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

File: templates/org/header.tmpl [hunk 1]
```diff
@@ -5,7 +5,8 @@
 				<div class="ui header">
 					<img class="ui image" src="{{.SizedRelAvatarLink 100}}">
 					<span class="text thin grey"><a href="{{.HomeLink}}">{{.DisplayName}}</a></span>
-
+					{{if .Visibility.IsLimited}}<div class="ui medium orange horizontal label" style="margin-left: 5px; margin-top: 2px">{{$.i18n.Tr "org.settings.visibility.limited_shortname"}}</div>{{end}}
```
Comment (6543) [inline templates/org/header.tmpl]: move css into less files (web_src/less)

**Review by 6543 — CHANGES_REQUESTED** [review]
I realy like how it looks like ...

two TODO's:
 * move inline css to less files
 * public repos in limited orgs are limited too

**Comment by smtaylorslc** [comment]
@6543 Moved styles to .less files in commit 17fbba7 . Re: repository "Limited" labels -- do we want to find some way to visually show that the "Limited" attribute/label is inherited from the org (opposed to the "Private" attribute, which is set at the repository level & also creates a label next to the repo name)?

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
🚀

**Comment by techknowlogick** [comment]
@6543 I think the 2nd one can be saved for later PR

**Pull request merged**